assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -65,6 +65,10 @@ function FramesLayer() {
     })
   );
 
+  const { isAnythingTransforming } = useTransform((state) => ({
+    isAnythingTransforming: state.state.isAnythingTransforming,
+  }));
```

Comment:
If you <kbd>esc</kbd> out of a transform (e.g. resize), this doesn't seem to reset:

![resize-esc-bug](https://user-images.githubusercontent.com/637548/112505169-e7312080-8d62-11eb-874e-ad782de6792b.gif)

Reply:
Looks like you've found an issue with `isAnythingTransforming`, congrats! 🎺  It's not being properly reset when `clearSelection` is happening.

I'm actually not sure now -- should the `Esc` just cancel the Moveable action? Or should it clear the selection fully as it's doing now 🤔  (Probably not to be changed within this PR though)

What we could do now is checking if the selection changes and nothing is selected anymore, we clear the transformations (if that's possible generally without ID, have to check). Otherwise, we can hide the design space if nothing is selected but that's not fixing the root issue.

Reply:
Should be fixed now in 129c19e.

Reply:
Wait, now I get an even weirder bug... where the frame is resized, but the display is not, so the frame and display are now mismatched:

![shape-frame-resize-abort-bug](https://user-images.githubusercontent.com/637548/112685396-e07dd880-8e4a-11eb-9f5c-12c686a36bbd.gif)

Reply:
Hm, looks like Moveable is not reset when this happens. Ugh, this is a bit more complicated issue than it seemed to be. Will investigate.

Reply:
Apparently using `Esc` while doing a transformation is buggy as it is:
The elements are deselected but Moveable transformation is not finished / cleaned up so it causes unexpected behaviour on `main`, too:
![escing](https://user-images.githubusercontent.com/3294597/113007692-b6902300-9176-11eb-96c7-852166f778b7.gif)

This will need its own issue but since no one has flagged it I suppose not a lot of people are using `Esc` while transforming something.